Lib/defconAppKit/controls/glyphCollectionView.py

```diff
@@ -253,9 +253,10 @@ def setMode(self, mode):
             self._glyphCellView.unsubscribeFromWindow()
         elif mode == "cell":
             documentView = self._glyphCellView
-            if placard is not None:
-                placard.button.set(0)
+#             if placard is not None:
+#                 placard.button.set(0)
         self._list.getNSScrollView().setDocumentView_(documentView)
+        self._list.getNSTableView().sizeToFit()
```

Comment:
The glyph names text in the list was getting truncated. Perhaps it belongs elsewhere, but when I tried putting it upstream, after line 219, the truncation still happened.